src/sentry/api/endpoints/release_thresholds/release_threshold_status_index.py

```diff
@@ -83,14 +88,29 @@ def validate(self, data):
 
 
 @region_silo_endpoint
+@extend_schema(tags=["Releases"])
 class ReleaseThresholdStatusIndexEndpoint(OrganizationReleasesBaseEndpoint, EnvironmentMixin):
     owner: ApiOwner = ApiOwner.ENTERPRISE
     publish_status = {
-        "GET": ApiPublishStatus.EXPERIMENTAL,
+        "GET": ApiPublishStatus.PUBLIC,
     }
 
+    @extend_schema(
+        operation_id="Retrieve status of a release threshold (Alpha)",
+        parameters=[GlobalParams.ORG_SLUG],
```

Comment:
This will include the query params in the docs. Note that they'll show up in the same order in the serializer, so you should reorder them to whatever makes the most sense
```suggestion
        parameters=[GlobalParams.ORG_SLUG, ReleaseThresholdStatusIndexSerializer],
```